game-core/src/main/java/games/strategy/engine/framework/ui/SaveGameFileChooser.java

```diff
@@ -55,7 +52,15 @@
 
     AUTOSAVE_EVEN(getEvenRoundAutoSaveFile(true));
 
-    private final File file;
+    private final Path path;
+
+    AUTOSAVE_TYPE(final File file) {
```

Comment:
Actually is there anything speaking against switching to Path for the calling code?
Path is obviously the more modern API.

Reply:
Nothing other than most of our legacy code still uses `File`, so that the `Path` would have to be converted to `File` eventually.  In this case, the value of `AUTOSAVE_TYPE#getFile()` is passed to `HeadlessGameServer#loadGameSave(File)`, which is then passed to `GameSelectorModel#load(File)`, which is then passed to `GameDataManager#loadGame(File)`.  You get the idea. :smile:

I think preferring `Path` over `File` is a Good Thing, in general, but it should probably be addressed in a series of separate PRs focused on that task.